src/renderer/pkijs/generatePems/common.ts

```diff
@@ -0,0 +1,85 @@
+import { fromBER } from 'asn1js'
+import { stringToArrayBuffer, fromBase64 } from 'pvutils'
+import fs from 'fs-sync'
```

Comment:
you also probably can't do this if you're going to import this in the browser - it doesn't have `fs` module